src/fhir_mcp_server/server.py

```
@@ -271,6 +273,19 @@ async def search(
+            List[str],
+            Field(
+                description="Specify FHIRPath expressions to filter the response to only the data you need, reducing token usage. Omitting returns the full resource.",
+                examples=[
+                    [
+                        "Condition.code",
+                        "Condition.clinicalStatus",
+                        "Bundle.link.where(relation='next').url",
+                    ]
+                ],
+            ),
+        ] = [],
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)